### PR TITLE
[DOC-2194] docs(Restpp): Update built-in-endpoints.adoc for restpp

### PR DESCRIPTION
**Pull request by ArthurXie-tg**

### CODE REVIEW

File: modules/API/pages/built-in-endpoints.adoc [hunk 1]
```diff
@@ -9,22 +9,22 @@ This page describes the REST API endpoints accessible on a TigerGraph server.
...
 This also applies to previously created cloud solutions that are resized on or after June 20, 2022. Use port 443 instead for API endpoint connectivity on cloud solutions:
 
-** Use API endpoint `443/restpp/<endpoint>` for RESTPP endpoints that were on port 9000
-
 ** Use API endpoint `443/<endpoint>` for endpoints that were on port 14240
 
-For all TigerGraph Server solutions not on TigerGraph Cloud, or for TigerGraph Cloud solutions created before June 20, 2022, ports 9000 and 14240 remain correct.
+For TigerGraph Cloud solutions created before June 20, 2022, ports 9000 and 14240 remain correct.
+
+In 4.1.0 and later versions, for all TigerGraph Server solutions not on TigerGraph Cloud, port 9000 are no longer exposed, please use `14240/restpp/v1/<endpoint>` instead.
```
Comment (dadongwang-tg) [inline modules/API/pages/built-in-endpoints.adoc]: "/restpp/v1" -> "/restpp"
after 4.1.1, we will add version

File: modules/API/pages/built-in-endpoints.adoc [hunk 2]
```diff
@@ -1117,265 +1117,9 @@ curl -X GET 'http://localhost:9000/deleted_vertex_check?threadnum=10&verbose=0&v
...
-(3) Otherwise, return a new token.
-
-|===
-
-==== Sample requests:
-The responses are slightly different between requests made with secrets and username-password pair.
-
-[tabs]
+As of 4.1.0, the use of plaintext tokens in authentication is deprecated.
+Use `/gsql/v1/tokens` trough 8123 port instead, refer to xref:gsql-ref:gsql-endpoints.adoc[] instead.
```
Comment (dadongwang-tg) [inline modules/API/pages/built-in-endpoints.adoc]: 8123 -> 14240

File: modules/API/pages/built-in-endpoints.adoc [hunk 3]
```diff
@@ -2915,7 +2658,7 @@ Request::
 --
 [source.wrap,console]
 ----
-curl --user tigergraph:tigergraph -X GET 'http://localhost:8123/gsql/queries/install/1234'
+curl --user tigergraph:tigergraph -X GET 'http://localhost:14240/gsql/v1/queries/install/1234'
```
Comment (dadongwang-tg) [inline modules/API/pages/built-in-endpoints.adoc:2661]: BTW, have we changed all 8123 to 14240 as well?

File: modules/API/pages/built-in-endpoints.adoc [hunk 4]
```diff
@@ -3773,7 +3516,7 @@ Request::
 [source.wrap,bash]
 ----
 curl -X POST --user tigergraph:tigergraph \
-  --url 'http://localhost:14240/gsql/library?functionName=gds.community.louvain&graph=poc_graph' \
+  --url 'http://localhost:14240/gsql/v1/gds.community.louvain?graph=poc_graph' \
```
Comment (dadongwang-tg) [inline modules/API/pages/built-in-endpoints.adoc]: should be `/gsql/v1/library/gds.community.louvain`?

Comment (ArthurXie-tg) [reply modules/API/pages/built-in-endpoints.adoc]: Done

File: modules/API/pages/built-in-endpoints.adoc (hunk 2 above)
Comment (ArthurXie-tg) [reply modules/API/pages/built-in-endpoints.adoc]: Done

File: modules/API/pages/built-in-endpoints.adoc (hunk 1 above)
Comment (ArthurXie-tg) [reply modules/API/pages/built-in-endpoints.adoc]: Done

File: modules/API/pages/built-in-endpoints.adoc (hunk 3 above)
Comment (ArthurXie-tg) [reply modules/API/pages/built-in-endpoints.adoc:2661]: We missed some places, let me change them

Comment (ArthurXie-tg) [reply modules/API/pages/built-in-endpoints.adoc:2661]: Done

**Review by arun-TG-PM — APPROVED** [review]

**Pull request merged**